packages/fluentui/react-northstar/src/themes/teams/siteVariables.ts

```diff
@@ -11,7 +11,10 @@ export { colors, contextualColors, naturalColors, primitiveColors, colorScheme,
 // BORDER STYLES
 //
 export const borderWidth = '1px';
-export const borderRadius = '2px';
+export const borderRadiusSmall = '2px'; // input focus states
+export const borderRadiusMedium = '4px'; // default border radius
+export const borderRadiusLarge = '6px';
```

Comment:
seems that `borderRadiusLarge` and `borderRadiusCircular` are never used in this PR?

Reply:
Removing these